function/function.go

```diff
@@ -137,6 +137,10 @@ func (f *Function) callAWSLambda(payload []byte) ([]byte, error) {
 	if err != nil {
 		if awserr, ok := err.(awserr.Error); ok {
 			switch awserr.Code() {
+			case "AccessDeniedException":
```

Comment:
@elsteelbrain I did it differently, please take a look

Reply:
Have you tested it?

Reply:
Yes, of course.